client/webpack.config.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
   output: {
     filename: 'app.js'
   },
+  devtool: 'source-map',
```

Comment:
I've added this - it only loads if you're in dev mode, which is useful on PROD and DEV.